Sources/Bugsnag/BugsnagConfig.swift

```diff
@@ -0,0 +1,39 @@
+import Vapor
+
+public struct BugsnagConfig {
+    public enum Error: Swift.Error {
+        case invalidConfig(field: String, Config)
+    }
+    
+    public let apiKey: String
+    public let notifyReleaseStages: [String]
+    public let endpoint: String
+    public let filters: [String]
+    public let stackTraceSize: Int
+    public let environment: Environment
+    
+    public init(_ config: Config) throws {
+        apiKey = try config.get("apiKey")
+        
+        notifyReleaseStages = try config["notifyReleaseStages"]?.array?.map {
+            guard let string = $0.string else {
+                throw Error.invalidConfig(field: "notifyReleaseStages", config)
```

Comment:
Would it be better to return an `Abort` error here with some meaningful description?